### PR TITLE
[Translation] added FileLoader.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

File: src/Symfony/Component/Translation/Loader/CsvFileLoader.php [hunk 1]
```diff
@@ -33,16 +31,8 @@ class CsvFileLoader extends ArrayLoader
      *
      * @api
      */
-    public function load($resource, $locale, $domain = 'messages')
+    public function loadResource($resource)
```
Comment (xabbuh) [inline src/Symfony/Component/Translation/Loader/CsvFileLoader.php]: Should be `protected` (like the method in the parent class).

File: src/Symfony/Component/Translation/Loader/CsvFileLoader.php [hunk 2]
```diff
@@ -33,16 +31,8 @@ class CsvFileLoader extends ArrayLoader
      *
      * @api
```
Comment (xabbuh) [inline src/Symfony/Component/Translation/Loader/CsvFileLoader.php]: should be removed

File: src/Symfony/Component/Translation/Loader/CsvFileLoader.php (hunk 1 above)
Comment (aitboudad) [inline src/Symfony/Component/Translation/Loader/CsvFileLoader.php]: fixed

**Comment by stof** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @aitboudad.

**Pull request merged**